apps/atomy-q/API/app/Http/Controllers/Api/V1/ApprovalController.php

```
@@ -132,6 +156,119 @@ public function show(Request $request, string $id): JsonResponse
+            provenance: $this->providerArtifactProvenance($summary, AiStatusSchema::ENDPOINT_GROUP_COMPARISON_AWARD),
+        );
+        $this->persistApprovalSummaryArtifact($comparisonRun, $approval, $artifact);
+
+        return response()->json([
+            'data' => [
+                'ai_summary' => [
+                    ...$artifact,
+                ],
+            ],
+        ]);
+    }
```

PHPMD (2.15.0)

[warning] The method summary() has 108 lines of code. Current threshold is set to 100. Avoid really long methods. (undefined)

(ExcessiveMethodLength)